docs/v3/source/includes/api_resources/_service_bindings.erb

```diff
@@ -2,15 +2,20 @@
 {
   "guid": "dde5ad2a-d8f4-44dc-a56f-0452d744f1c3",
   "type": "app",
```

Comment:
This is the file that refers to the existing implementation of v3 service bindings. The one you want to edit is `docs/v3/source/includes/api_resources/_service_credential_bindings.erb`